assets/src/components/parts/janus-mcq/MultipleChoiceQuestion.tsx

```diff
@@ -828,7 +828,11 @@ const MultipleChoiceQuestion: React.FC<PartComponentProps<McqModel>> = (props) =
   }
 
   return ready ? (
-    <div data-janus-type={tagName} style={styles} className={`mcq-input`}>
+    <div
+      data-janus-type={tagName}
+      style={styles}
+      className={`mcq-input ${customCssClass === 'no-wrap' ? customCssClass : ''}`}
```

Comment:
Need to add custom class if it is 'no-wrap'. Rest of the classes already gets applied on custom component. I am not sure if we can add 'no-wrap' class from override file. @bsparks  thoughts?

Reply:
is it not possible to adjust via css using something like `janus-mcq.no-wrap .mcq-input {`

Reply:
Done. I will update the CSS override file